message_ix_models/tools/costs/projections.py

```diff
@@ -477,67 +489,115 @@ def create_message_outputs(
     )
 
     dtypes.update(year_act=int)
-    fom = (
-        df_merge.copy()
-        .drop(columns=["inv_cost"])
-        .assign(key=1)
-        .merge(
-            pd.DataFrame(data={"year_act": config.seq_years}).assign(key=1),
-            on="key",
-        )
-        .drop(columns=["key"])
-        .query("year_act >= year_vtg")
-        .assign(
-            val=lambda x: np.where(
-                x.year_vtg <= y_base,
-                np.where(
-                    x.year_act <= y_base,
-                    x.fix_cost,
+
+    if config.use_vintages is True:
```

Comment:
```suggestion
    if config.use_vintages:
```
Not a necessary change, but since this is a bool, `if config.use_vintages` will evaluate to `True` if and only if this setting is `True` and `False` iif it's `False`.